src/models/properties.rs

```diff
@@ -247,8 +248,9 @@ pub struct FileReference {
 
 #[derive(Serialize, Deserialize, Debug, Eq, PartialEq, Clone)]
 #[serde(tag = "type")]
+//#[serde(untagged)]
```

Comment:
unneeded?

Reply:
Yes, unneded. Was experimenting with parsing errors and forgot to remove. Will remove in next commit